### PR TITLE
TINKERPOP-2150 Support configuring the registry builder in config

### PR DESCRIPTION
**Pull request by jorgebay**
https://issues.apache.org/jira/browse/TINKERPOP-2150

VOTE +1

### CODE REVIEW

**Comment by spmallette** [comment]
Please add an entry to CHANGELOG when you merge but otherwise VOTE +1

File: gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/ser/GraphBinaryMessageSerializerV1.java [hunk 1]
```diff
@@ -42,6 +43,7 @@
 
 public class GraphBinaryMessageSerializerV1 extends AbstractMessageSerializer {
     public static final String TOKEN_CUSTOM = "custom";
+    public static final String TOKEN_BUILDER = "builder";
```
Comment (dkuppitz) [inline gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/ser/GraphBinaryMessageSerializerV1.java]: The new configuration option should be added to the [docs](https://github.com/apache/tinkerpop/blob/TINKERPOP-2150/docs/src/reference/gremlin-applications.asciidoc#graphbinary).

Comment (jorgebay) [reply gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/ser/GraphBinaryMessageSerializerV1.java]: Nice catch @dkuppitz , will do!

**Comment by jorgebay** [comment]
I've updated the docs and added an entry to the CHANGELOG.

**Comment by dkuppitz** [comment]
VOTE +1

**Pull request merged**